Backend/Remora.Discord.Core/Optional.cs

```diff
@@ -136,5 +121,13 @@ public override int GetHashCode()
         {
             return HashCode.Combine(_value, this.HasValue);
         }
+
+        /// <inheritdoc />
+        public override string ToString()
+        {
+            return this.HasValue
+                ? $"{{{_value?.ToString() ?? "null"}}}"
```

Comment:
What's with the string interpolation here?

Reply:
Uhh... it's a way to build strings?

Reply:
Yes, but why :P What's the point of adding the curly braces around the value?

Reply:
It provides a lot of subtle visual clues about the fact that this isn't just a "pure" value, but it's wrapped in a container. Pretty darned minor, but I like the idea. For example, it provides a distinction between an empty optional value, and an optional value containing the word "Empty". Also, from a live use-case...

![Capture](https://user-images.githubusercontent.com/26645431/126030969-3e252f53-281f-475a-98eb-21d74f0f2308.PNG)


Here, it also provides a subtle indication that some of these logged values COULD have been unspecified, but weren't.

Reply:
Fair enough!